Shared/View/AuthenticationView.swift

```diff
@@ -65,13 +75,18 @@ struct SignUpView: View {
                     do {
                         try await user.signUp(email: email, username: username, password: password)
                     } catch {
-                        // TODO: display error to user
-                        print(error)
+                        alertText = error.localizedDescription
+                        showingAlert = true
                     }
                 }
             } label: {
                 Text("Sign Up")
             }
+            .alert("Error signing up", isPresented: $showingAlert) {
```

Comment:
@elulu3 thanks for adding the error alerts to handle some of the errors! I tweaked them a little bit to show the more detailed error messages in addition to error titles, allowing us to display "user already exist", "incorrect email format", and etc.